src/wallet/wallet.h

```diff
@@ -961,6 +961,7 @@ class CWallet final : public CCryptoKeyStore, public CValidationInterface
     isminetype IsMine(const CTxOut& txout) const;
     CAmount GetCredit(const CTxOut& txout, const isminefilter& filter) const;
     bool IsChange(const CTxOut& txout) const;
+    bool IsChange(const CScript& dest) const;
```

Comment:
Please make the naming between those two consistent:

```
bool CWallet::IsChange(const CScript& script) const { … }
bool IsChange(const CScript& dest) const;
```

Make both `script` or both `dest` :-)

Reply:
Done.